src/App.tsx

```diff
@@ -1,14 +1,13 @@
 import GlobalStyles from './GlobalStyles'
+import { MainStyle } from './AppStyles'
 import Routers from './router'
 
 function App() {
   return (
-    <div>
-      <section className='max-w-screen-sm mx-auto min-h-screen'>
-        <GlobalStyles />
-        <Routers />
-      </section>
-    </div>
+    <MainStyle>
+      <GlobalStyles />
+      <Routers />
+    </MainStyle>
```

Comment:
```suggestion
<>
      <Routers />
 </>
```

app.jsx 에서는 라우터만 관리하는 것이 좋습니당 ◡̈  
GlobalStyles는 `src/index.tsx` 파일로 이동하는 게 좋을 것 같습니다.

```
// src/index.tsx
  <React.StrictMode>
    <BrowserRouter>
      <GlobalStyles />
      <App />
    </BrowserRouter>
  </React.StrictMode>,
```